.github/CODEOWNERS

```diff
@@ -1,106 +1,586 @@
+
+# ServiceLabel: %Commerce %Service Attention
+#/<NotInRepo>/            @ms-premp @qiaozha
 
 # PRLabel: %Communication
 /sdk/communication/                             @RezaJooyandeh @DominikMe @Azure/azure-sdk-communication-code-reviewers
 
-/sdk/cognitiveservices/language.textanalytics/  @assafi 
-/sdk/cognitiveservices/vision.computervision/   @toothache
-/sdk/cognitiveservices/vision.face/             @TFR258 
+# ServiceLabel: %Compute %Service Attention
+# PRLabel: %Compute
+/sdk/compute/                                   @bilaakpan-ms @sandido @dkulkarni-ms @haagha @madewithsmiles @MS-syh2qs @grizzlytheodore @amjads1 @avirishuv @vaibhav-agar @Drewm3
+
+# ServiceLabel: %Compute - Extensions %Service Attention
+#/<NotInRepo>/            @amjads1 @Drewm3
+
+# ServiceLabel: %Compute - Images %Service Attention
+#/<NotInRepo>/            @Drewm3 @vaibhav-agar
+
+# ServiceLabel: %Compute - Managed Disks %Service Attention
+#/<NotInRepo>/            @Drewm3 @vaibhav-agar
+
+# ServiceLabel: %Compute - RDFE %Service Attention
+#/<NotInRepo>/            @Drewm3 @avirishuv
+
+# ServiceLabel: %Compute - VM %Service Attention
+#/<NotInRepo>/            @Drewm3 @avirishuv
+
+# ServiceLabel: %Compute - VMSS %Service Attention
+#/<NotInRepo>/            @Drewm3 @avirishuv
+
+# ServiceLabel: %Consumption %Service Attention
+/sdk/consumption/Microsoft.Azure.Management.Consumption/            @ms-premp
+
+# ServiceLabel: %Connected Kubernetes %Service Attention
+#/<NotInRepo>/            @akashkeshari
+
+# ServiceLabel: %Container Instances %Service Attention
+/sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/            @dkkapur
+
+# ServiceLabel: %Container Registry %Service Attention
+/sdk/containerregistry/            @toddysm @yugangw-MSFT
+
+# ServiceLabel: %Cosmos %Service Attention
+/sdk/cosmosdb/          @Wmengmsft @MehaKaushik @shurd @anfeldma-ms @zfoster @kushagraThapar @southpolesteve
+
+# ServiceLabel: %Cost Management %Service Attention
+/sdk/cost-management/Microsoft.Azure.Management.CostManagement/           @ms-premp @ramaganesan-rg
+
+# ServiceLabel: %Customer Insights %Service Attention
+/sdk/customer-insights/Microsoft.Azure.Management.CustomerInsights/            @shefymk
+
+# ServiceLabel: %CycleCloud %Service Attention
+#/<NotInRepo>/            @adriankjohnson
+
+# ServiceLabel: %Data Bricks %Service Attention
+#/<NotInRepo>/            @yagupta
+
+# ServiceLabel: %DataBox %Service Attention
+/sdk/databox/Microsoft.Azure.Management.DataBox/            @tmvishwajit @matdickson @manuaery @madhurinms
+
+# ServiceLabel: %Data Catalog %Service Attention
+#/<NotInRepo>/            @anilman
+
+# ServiceLabel: %Data Factory %Service Attention
+/sdk/datafactory/Microsoft.Azure.Management.DataFactory/            @shawnxzq @lmy269
+
+# ServiceLabel: %Data Lake %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Storage Gen1 %Service Attention
```

Comment:
Is the distinction between Gen1 and Gen2 germane to routing?  As triage, I'm often unclear on the generation and they route to the same contacts.   Maybe consider removing these and collapsing into  the "Data Lake Store" label instead?

Reply:
No -- the gen1 is a thing about Data Lake.... They have 2 gens.

Reply:
I know they have two generations.... but they both end up going to the same team.   Making the distinction during triage is often a "well, Gen2 because.... reasons!" decision.    Figured it may be worth collapsing.